### PR TITLE
Encode ITU-WHO T-FG-AI4H document

### PR DESCRIPTION
**Pull request by anermina**
Notes:
- "References" title has been changed to "Bibliography", because "References" is used for normative references, which cannot have a numeric identifier in ITU flavor. All the references here have a numeric identifier.
- Opened tickets:
https://github.com/metanorma/metanorma-itu/issues/333
https://github.com/metanorma/metanorma-itu/issues/334
https://github.com/metanorma/metanorma-itu/issues/335
https://github.com/metanorma/metanorma-itu/issues/336
https://github.com/metanorma/metanorma-itu/issues/337
https://github.com/metanorma/metanorma-itu/issues/338
https://github.com/metanorma/metanorma-itu/issues/339

### CODE REVIEW

File: sources/T-FG-AI4H-2022-MSW-E.adoc [hunk 1]
```diff
@@ -0,0 +1,7756 @@
...
+:docnumber: FG-AI4H DEL01
+:series: Focus Group on Artificial Intelligence for Health (FG-AI4H)
+:published-date: 2022-06-02
+:copyright-year: 2022
+:edition: 1
+:status: in-force
+:recommendationnumber: ITU-T FG-AI4H
+:imagesdir: images
+:doctype: focus-group
+:docfile: T-FG-AI4H-2022-MSW-E.adoc
```
Comment (ronaldtse) [inline sources/T-FG-AI4H-2022-MSW-E.adoc]: @anermina we no longer need the `docfile` attribute, and it is deprecated, please always remove it. thanks!

File: sources/T-FG-AI4H-2022-MSW-E.adoc [hunk 2]
```diff
@@ -0,0 +1,7756 @@
...
+:affiliation_2: World Health Organization
+:role_2: editor
+:email_2: pujaris@who.int
+:keywords: Artificial intelligence; AI for health; Ethics; Governance
+:toclevels: 3
+:toc-figures: true
+:mn-document-class: itu
+:mn-output-extensions: xml,html,doc,pdf,rxl
+:local-cache-only:
+:data-uri-image:
```
Comment (ronaldtse) [inline sources/T-FG-AI4H-2022-MSW-E.adoc]: Let's remove `data-uri-image` because it is now enabled by default. Thanks!

**Review by ronaldtse — CHANGES_REQUESTED** [review]
There are two attributes to remove, and can we split this file into separate sections? Thanks!

**Comment by anermina** [comment]
Thank you @ronaldtse! Changes have been applied.

I addressed several new tickets in the meanwhile. They are listed in the notes of this PR.
However, there is still one unclarity regarding the front page, i.e. document identifiers which should be used for `:doctype: focus-group`. Currently, we have the following differences (one regarding the `series` attribute has been addressed in a ticket):
image

**Comment by ronaldtse** [comment]
Thank you @anermina ! Let's merge this first and then keep the pending tasks as a separate issue.

**Review by ronaldtse — APPROVED** [review]

**Comment by bquast** [comment]
Dear colleagues, thank you very much. I will distribute it to our delegates.

**Comment by ronaldtse** [comment]
Thanks @bquast 👍

**Pull request merged**